document/core/syntax/modules.rst

```diff
@@ -354,7 +354,7 @@ The |MIMPORTS| component of a module defines a set of *imports* that are require
      \IDGLOBAL~\globaltype \\
    \end{array}
 
-Each import is identified by a two-level :ref:`name <syntax-name>` space, consisting of a |IMODULE| name and a unique |INAME| for an entity within that module.
+Each import is identified by a two-level :ref:`name <syntax-name>` space, consisting of a |IMODULE| name and a |INAME| for an entity within that module.
```

Comment:
I also suggest changing "is identified by" to "utilizes", or something, because two imports with the same names may still have distinct identities: they have different indices and/or kinds, and they may be resolved to different entities within the same instantiation.

Reply:
Changed to "labeled".